weeb/module.py

```diff
@@ -0,0 +1,143 @@
+import discord
+from discord.ext import commands
+from discord.abc import PrivateChannel
+import aiohttp
+from typing import Optional
+from core import utils, text, logging, i18n
+
+_ = i18n.Translator("modules/fun").translate
+
+
+class Weeb(commands.Cog):
+    """
+    Weeby commands for your own Weebiness
+
+    Includes:
+        ?sauce that will give your desired sauce if you supply it with a magic number
+    """
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @classmethod
+    def doujin_tags(cls, dic):
+        characters, parodies, tags, artists, groups, languages, categories = (
+            [],
+            [],
+            [],
+            [],
+            [],
+            [],
+            [],)
+        tags = []
+        for tag in dic["tags"]:
+            if tag["type"] == "character":
+                characters.append(tag["name"])
+            elif tag["type"] == "parody":
+                parodies.append(tag["name"])
+            elif tag["type"] == "tag":
+                tags.append(tag["name"])
+            elif tag["type"] == "artist":
+                artists.append(tag["name"])
+            elif tag["type"] == "group":
+                groups.append(tag["name"])
+            elif tag["type"] == "language":
+                languages.append(tag["name"])
+            elif tag["type"] == "category":
+                categories.append(tag["name"])
+
+        characters = ["Characters", ",".join(str(e) for e in characters)]
+        parodies = ["Parodies", ",".join(str(e) for e in parodies)]
+        tags = ["Tags", ",".join(str(e) for e in tags)]
+        artists = ["Artists", ",".join(str(e) for e in artists)]
+        groups = ["Groups", ",".join(str(e) for e in groups)]
+        languages = ["Languages", ",".join(str(e) for e in languages)]
+        categories = ["Categories", ",".join(str(e) for e in categories)]
```

Comment:
So, the tag names should not be translated?

Reply:
Since the tags itself are not translated i thought it would be redundant but it can be done.

Reply:
@sinus-x by tags you mean the embed field titles or the tags in the "Tags" field? Because that would be an incredible amount of work. If it's just about the titles I'm fine with it otherwise that would not be recommended.